Formula/pipx.rb

```diff
@@ -3,6 +3,7 @@ class Pipx < Formula
   homepage "https://github.com/pipxproject/pipx"
   url "https://files.pythonhosted.org/packages/c3/9e/5d93d784225a36c36d8be09144eea0a73f9f85385dfa02881c51644495be/pipx-0.15.4.0.tar.gz"
   sha256 "b90285a1c6eddf45ba4e48110b81d7c1e285b127e82fa99a61168937f98e5cb5"
+  license "MIT"
```

Comment:
[`BSD-3-Clause` also?](https://github.com/pipxproject/pipx/blob/2f31607b04bf5176a5b4c4d397ad1274d30f7a18/LICENSE#L27-L54)

Reply:
They put in the `authors.txt`, https://github.com/pioneerspacesim/pioneer/blob/master/AUTHORS.txt#L127-L132

Also, this is a good case, the different parts of the codebase got licensed under different licenses.

Reply:
adding the comment in the wrong place 😅

Reply:
Another way to look at this license thing is 2014 is licensed under BSD-x, and 2018, it got relicensed under MIT. So now this is MIT. 

This would be a challenge for us to track down the license change in the future.

Reply:
But according to law, the license change also needs to get all approvals or consensus from the contributors, which makes the license change rarely happen though.

Reply:
👍